index.html

```diff
@@ -169,6 +165,30 @@ <h2>Producing</h2>
         </p>
       </li>
 
+
+      
+
+      <li>
+        <p>
+          <a data-cite="CWT_HEADER#section-3">iss</a> MAY be present in <a data-cite="RFC9052#name-common-cose-header-paramete">COSE Headers</a>, 
+          and MUST match the <a data-cite="DID-CORE#verification-methods">controller</a> of the dereferenced <a data-cite="DID-CORE#dfn-publickeyjwk">publicKeyJwk</a>.
+        </p>
+      </li>
+
+      <li>
+        <p>
+          <a data-cite="RFC9052#section-3.1">alg</a> MUST be present in <a data-cite="RFC9052#name-common-cose-header-paramete">COSE Headers</a>, 
```

Comment:
I think to avoid mistakes it should explicitly say in which buckets each parameter may go. JWS compact only has a protected header, but COSE always has both unprotected and protected. For simplicity and alignment with JWS compact, it's probably ok to say that all have to be in protected, even though some would argue that `kid` may be unprotected.

Reply:
tell me what you want the `protected` and `unprotected` header to look like :)

So far, I put everything in protected, to align with JWS.

I will make that more explicit, unless you disagree.

Reply:
https://github.com/transmute-industries/vc-cose/issues/9